### ISSUE TITLE
Extra line breaks added to Redactor fields

### ISSUE THREAD

**Issue by alexroper** [verbb/vizy#100]
**Description**

When using a Redactor field inside a Vizy block, html block elements will have double line breaks between them after saving the entry. The problem does not appear in the same Redactor field used outside of Vizy.

Before saving:

![Screen Shot 2022-02-09 at 2 50 38 PM](https://user-images.githubusercontent.com/377542/153303980-98246a71-fa49-4d4b-8693-7cf3914d8428.png)

After saving:

![Screen Shot 2022-02-09 at 2 51 00 PM](https://user-images.githubusercontent.com/377542/153303994-d6aa49c1-2caa-4230-8777-1c6f448d6ff2.png)

The HTML looks clean as expected without any extra elements or strange characters:

![Screen Shot 2022-02-09 at 2 55 37 PM](https://user-images.githubusercontent.com/377542/153304477-4de8fe97-4623-4364-b684-92ea5df31a76.png)

I think I tracked the problem down to this CSS:

```css
.ProseMirror [contenteditable="false"] [contenteditable="true"] {
    white-space: pre-wrap;
}
```

Disabling `white-space: pre-wrap` in dev tools fixes the problem.

**Steps to reproduce**
1. Create a Vizy block with a redactor field inside.
2. Create an entry and add multiple lines to the redactor field.
3. Save the entry.
4. Edit the entry again. There will be double line breaks between html block elements instead of single line breaks.

**Additional info**
- Plugin version: 1.0.9
- Craft version: Craft Pro 3.7.32
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.10](https://github.com/verbb/vizy/releases/tag/1.0.10)
